agent/engine/image/types.go

```diff
@@ -60,6 +60,12 @@ func (imageState *ImageState) AddImageName(imageName string) {
 	}
 }
 
+func (imageState *ImageState) GetCountImageNames() int {
```

Comment:
The same as I mentioned to `GetCountImageStates`, what about making it more general like: `AllImages`?